VERSION

```diff
@@ -1 +1 @@
-0.5.0
\ No newline at end of file
+0.5.1
```

Comment:
Since it's a minor change, figured this was the correct path for the next version number. Happy to change to whatever.